src/Appium.Net/Appium/ImageComparison/FeaturesMatchingOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace OpenQA.Selenium.Appium.ImageComparison
+{
+    public class FeaturesMatchingOptions : ComparisonOptions
+    {
+        public string DetectorName { get; set; }
+        public string MatchFunc { get; set; }
+        public int GoodMatchesFactor { get; set; }
+
+        public override Dictionary<string, object> GetParameters()
+        {
+            return new Dictionary<string, object> {
+                { "detectorName", DetectorName },
```

Comment:
an option should only be included into the request if its value has been explicitly changed